.appveyor.yml

```diff
@@ -74,11 +74,23 @@ after_test:
 cache:
   - '%R_LIB_PATH%'
 
+on_failure:
+  - 7z a failure.zip *.Rcheck\*
```

Comment:
I guess it should be `R-package\*.Rcheck\*`.

Reply:
In the [.appveyor.yml](https://github.com/RGF-team/rgf/blob/R-package-version-update/.appveyor.yml) file you specified in the same way the *python-package* folder path however there are no artifacts created in case of a successful build.
In my opinion, we have to recursively search the disk for files or folders which have the specific naming ('.Rcheck' or 'dist'). In the [appveyor documentation](https://www.appveyor.com/docs/packaging-artifacts/#basics) is stated how files or folders should be recursively searched on disk. 
Therefore if the absolute path is

```bash
'C:/projects/rgf/R-package/RGF.Rcheck/00check.log'
```
then we probably have to search for the *Rcheck* folder in the following way

```bash
**\*.Rcheck\*
```
However, what bothers me in the failed build the most is the following,

```bash
Scanning the drive:
0 files, 0 bytes
```
Let me know what you think.

Reply:
We know where the needed files are exactly placed. Why should we complicate the task where it's not needed (it also will increase the CI time)?

Reply:
I modified the path as I sugested earlier, and here is the artifact: https://ci.appveyor.com/project/RGF-team/rgf/builds/26776638/artifacts. Can you please take a look?

Reply:
thanks for taking the initiative